client/post-editor/editor-categories-tags/accordion.jsx

```diff
@@ -93,37 +79,29 @@ module.exports = React.createClass( {
 				/>
 			</AccordionSection>
 		);
-	},
+	}
 
 	getCategoriesSubtitle() {
-		const siteId = this.props.site.ID;
-
-		// There's also post.categories but it doesn't have category names
-		// either, so we need to get them from the site categories data
-		const categories = this.getCategories().map( ( categoryId ) => {
-			const category = TermStore.get( siteId, categoryId );
-			return category ? unescapeString( category.name ) : null;
-		} ).filter( categoryName => categoryName );
+		const { translate, postTerms, defaultCategory } = this.props;
+		const categories = toArray( get( postTerms, 'category' ) );
```

Comment:
`toArray` will always return an array, so `size` (and perhaps the variable assignment) on the following line seem unnecessary in lieu of a simple `categories.length` property access.